apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {Table} from 'reactabular';
+import color from "../../util/color";
+
+const styles = {
+  cell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+  },
+  nameCell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+    fontFamily: '"Gotham 5r", sans-serif',
+  },
+  headerCell: {
+    border: '1px solid gray',
+    padding: 10,
+    backgroundColor: color.teal,
+  },
+};
+
+/**
+ * Map from project type to friendly name.
+ * @type {Object}
+ */
+const PROJECT_TYPE_MAP = {
+  applab: 'App Lab',
+  gamelab: 'Game Lab',
```

Comment:
I don't think we want to pass localized dashboard strings down to the app, so I've just added new localized strings for these in `apps/`. I'd be open to putting this in a more centralized location, however I'm tempted to wait until we need it somewhere else before doing so.